tests/cases/project/realisticNpmTypes.json

```diff
@@ -0,0 +1,10 @@
+{
+    "scenario": "Realistic npm package types",
+    "projectRoot": "tests/cases/projects/realisticNpmTypes",
+    "inputFiles": [
+        "index.ts"
+    ],
+    "declaration": true,
+    "baselineCheck": true,
+    "moduleResolution": "node"
+}
```

Comment:
'doh. It's just added by default in so many places that I thought this may be one of them. Added and fixed.